### DISCUSSION TITLE
How to achieve fully transparency in ClientRect area?

### DISCUSSION THREAD

**Discussion by LigaLiao** [microsoft/WindowsAppSDK#1448] [Q&A] [answered]
In the case of using MFC：SetLayeredWindowAttributes(RGB(255, 255, 255), 0, LWA_COLORKEY);
but, in the WinUI, This doesn't work.

**Comment by btueffers** [comment, marked as answer]
Hi @LigaLiao, transparent windows are not currently supported. If you'd like to make a feature request for this please create a new Issue!
